frontend/src/Routes/index.ts

```diff
@@ -120,6 +123,23 @@ export const otherRoutes: Routes = {
   },
 };
 
+const laboratoryRoutes: Routes = {
+  createLaboratory: {
+    path: '/laboratorios/nuevo',
+    view: LaboratoryCreateForm,
+    verboseName: 'Registrar Laboratorio',
+    showInMenu: true,
+    hasAccess: ((auth) => auth.isAdmin),
+  },
+  listLaboratory: {
+    path: '/laboratorios',
+    view: ListLaboratory,
+    verboseName: 'Laboratorios',
+    showInMenu: true,
+    hasAccess: ((auth) => auth.isClient),
```

Comment:
Estaria bien que el admin pudiera ver los laboratorios tamiben.
```suggestion
    hasAccess: ((auth) => auth.isClient || auth.isAdmin),
```